LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2021 Luca Casonato
+Copyright (c) 2021-2023 Luca Casonato
```

Comment:
Judging by the title, I feel like this is out of the scope of this PQ.

Reply:
Agreed. I can revert, I just don't think this is worth a separate PR.

Reply:
You can then update your title.

Reply:
Maybe to `chore: bump dependencies and license year`.